.github/workflows/pull-request.yml

```diff
@@ -2,7 +2,6 @@ name: Pull Request
 
 on:
   pull_request:
-    branches: '*'
```

Comment:
this is broken yml syntax and prevents the pr workflow from running (only sometimes which is very confuse)